librarycard.py

```diff
@@ -541,4 +548,270 @@ async def leaderboard(ctx):
   pagination = Paginator(pages=pages)
   await pagination.respond(ctx.interaction)
 
+@bot.slash_command(name="start-session", description = "Starts a new reading session for your Flight")
+@guild_only()
+@default_permissions(manage_messages=True)
+async def startSession(ctx):
+  search = {
+    'guild': ctx.guild.id,
+    'ended': {'$exists': False}
+  }
+
+  # Caps so the server can only have one active session at a time.
+  found = nominateSessions.count_documents(search, limit=1)
+  if(found == 1):
```

Comment:
The `0` condition is the positive result, so any deviation from 0 returned rows should trigger this code block rather than explicitly checking for a single returned row.  MongoDB may have a bad day and not respect the limit, or some other cosmic bug may occure.
Consider `!= 0` instead of `== 1`